### PR TITLE
Incorrect restriction in  withdrawls edit form

### PR DESCRIPTION
**Pull request by slavcho**
The amount of the withdrawals can be a decimal number. 
It should not be limited to integers only.

Before:

image

### CODE REVIEW

**Comment by sashko9807** [comment]
If I remember correctly ,we save the amount of a withdrawal record in stotinki as an integer, and conversions to decimals are only done on the front end for data visualization.

**Comment by slavcho** [comment]
> If I remember correctly ,we save the amount of a withdrawal record in stotinki as an integer, and conversions to decimals are only done on the front end for data visualization.

this is handled in:
```
  function handleSubmit(values: WithdrawalInput) {
    const data: WithdrawalData = {
      amount: toMoney(values.amount),
      

```

Actually we can save the form, but we cannot edit it later. It should be the same logic.
And this restriction is not in the CreateForm.

**Comment by slavcho** [comment]
@ani-kalpachka Can you please review this since Diana requested it urgently?

File: src/components/admin/withdrawals/EditForm.tsx [hunk 1]
```diff
@@ -47,7 +47,7 @@ const validationSchema: yup.SchemaOf<WithdrawalData> = yup
   .defined()
   .shape({
     status: yup.string().trim().min(1).max(10).required(),
-    amount: yup.number().positive().integer().required(),
+    amount: yup.number().positive().required(),
```
Comment (sashko9807) [inline src/components/admin/withdrawals/EditForm.tsx:50]: Not that it is an issue considering the amount field can't be edited in EditForm, but maybe for consistency reason we should at least have the number().positive(), rules for amount validation.

**Review by sashko9807 — APPROVED** [review]

**Pull request merged**